File: data/CodeLlama/readable_format_HPEC/fortran/file116.f90
program DRB080_func_arg_orig_yes
    use omp_lib
    use global
    implicit none

    integer :: i
    i = 0

    !$omp parallel
    call f1(i)
    !$omp end parallel

    if (i /= 0) then
        print 100, i
        100 format ('i =',i3)
    end if
end program

File: data/llama3_8B_ins_results/readable_format_HPEC/cpp/file116.cpp
#include <iostream>
#include <omp.h>

using namespace std;

int main() {
    int i = 0;

    #pragma omp parallel
    {
        f1(i);
    }

    if (i!= 0) {
        cout << "i = " << i << endl;
    }

    return 0;
}
